### PR TITLE
Add: slider

### PR DESCRIPTION
**Pull request by gauravsinhaweb**

### CODE REVIEW

File: components/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.slider-pixel:hover {
+  opacity: 1;
+}
+
+.slider-pixel::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 20px;
```
Comment (mehulsatardekar) [inline components/slider/slider.css]: use rem here

File: components/slider/slider.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  height: 20px;
+  background: var(--btn-yellow);
+  cursor: pointer;
+  border-radius: 50%;
+}
+
+.slider-pixel::-moz-range-thumb {
+  width: 25px;
```
Comment (mehulsatardekar) [inline components/slider/slider.css]: use rems here too

**Review by mehulsatardekar — APPROVED** [review]
good to merge the branch after the changes

File: components/slider/slider.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+.slider-pixel:hover {
+  opacity: 1;
+}
+
+.slider-pixel::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 20px;
+  height: 20px;
```
Comment (JuvitaSaini09) [inline components/slider/slider.css]: u could have used rem

File: components/slider/slider.css (hunk 2 above)
Comment (JuvitaSaini09) [inline components/slider/slider.css]: here too

**Review by JuvitaSaini09 — APPROVED** [review]
It looks good to merge

File: components/slider/slider.css (hunk 1 above)
Comment (gauravsinhaweb) [reply components/slider/slider.css]: Update the changes. Thanks!

File: components/slider/slider.css (hunk 2 above)
Comment (gauravsinhaweb) [reply components/slider/slider.css]: Update the changes. Thanks!

File: components/slider/slider.css (hunk 3 above)
Comment (gauravsinhaweb) [reply components/slider/slider.css]: Update the changes. Thanks!

File: components/slider/slider.css (hunk 2 above)
Comment (gauravsinhaweb) [reply components/slider/slider.css]: Update the changes. Thanks!

**Pull request merged**